pagopa/io-app-design-system
# [IOPID-953] add notification pictogram

## Short description
In order to realize [this screen](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?node-id=2843%3A27915&mode=dev) i need to insert into the design system the notification pictogram

## List of changes proposed in this pull request
- Add notification pictogram
<img width="431" alt="Screenshot 2023-10-09 alle 09 21 07" src="https://github.com/pagopa/io-app-design-system/assets/83651704/343987f1-c51b-4659-b83d-f7c367c5c278">


## How to test
Run the example app and navigate into the pictogram page to see the new pictogram named as "notification"
